app/assets/javascripts/discourse/tests/unit/controllers/user-notifications-test.js

```diff
@@ -58,29 +56,4 @@ module("Unit | Controller | user-notifications", function (hooks) {
 
     assert.strictEqual(markRead, true);
   });
-
-  test("Shows modal when has high priority notifications", function (assert) {
```

Comment:
This test quite messy... a stub, fake, and a send on a controller is just too much happening in one place. Aditionally, this logic is already tested across the multiple different notification types so there is no point in keeping this.